### DISCUSSION TITLE
Default locale

### DISCUSSION THREAD

**Discussion by Joseph-Naveen** [js-fns/date-fns#3679] [Q&A] [answered]
Is any locale format available for default 'en' not ('enUS', 'enIN', 'enNZ'...)?

**Comment by LinThitHtwe** [comment]
I think the default is already en, you don't need to specify it 

```javascript
formatRelative(subDays(new Date(), 3), new Date());
//=> "last Friday at 7:26 p.m."

**Reply by Joseph-Naveen** [reply]
Can I import like this - `import { de, ko, en } from 'date-fns/locale';
`
![image](https://github.com/date-fns/date-fns/assets/54658902/65c8daf9-597a-42c3-b790-1f3369b0f280)

**Reply by LinThitHtwe** [reply]
how about like this instead 
```javascript
import en from 'date-fns/locale/en';
import de from 'date-fns/locale/de';
import ko from 'date-fns/locale/ko';
```

**Reply by Joseph-Naveen** [reply]
I am getting an error like this for 'en' - Unable to resolve the path to module 'date-fns/locale/en'.

For other imports(de, ko) it works fine

**Reply by LinThitHtwe** [reply, marked as answer]
It seems that the 'en' locale is not available. However, you can use one of the following specific English locales:

- `en-AU`
- `en-CA`
- `en-GB`
- `en-IE`
- `en-IN`
- `en-NZ`
- `en-US`
- `en-ZA`
